src/api/custom-resolvers/mutations/update-loss-payee-financial-details-uk/index.ts

```diff
@@ -0,0 +1,55 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import encrypt from '../../../helpers/encrypt';
+import { ApplicationLossPayeeFinancialUk, SuccessResponse } from '../../../types';
+
+/**
+ * updateLossPayeeFinancialDetailsUk
+ * encrypts sortCode and accountNumber
+ * saves sortCode, sortCodeVector, accountNumber, accountNumberVector, bankAddress to db
+ * @param {Object} GraphQL root variables
+ * @param {ApplicationLossPayeeFinancialUk} GraphQL variables for the ApplicationLossPayeeFinancialUk mutation
+ * @param {Object} KeystoneJS context API
+ * @returns {Object} Object with success flag
+ */
+const updateLossPayeeFinancialDetailsUk = async (root: any, variables: ApplicationLossPayeeFinancialUk, context: Context): Promise<SuccessResponse> => {
+  try {
+    console.info('Updating loss payee financial details UK %s', variables.id);
+
+    const { id, accountNumber, sortCode, bankAddress } = variables;
+
+    /**
+     * object with encrypted data and initialisation vectors
+     * encrypts accountNumber and sortCode
+     * adds the initialisation vectors
+     */
+    const updateData = {
+      accountNumber: encrypt(accountNumber).value,
+      accountNumberVector: encrypt(accountNumber).iv,
+      sortCode: encrypt(sortCode).value,
+      sortCodeVector: encrypt(sortCode).iv,
+      bankAddress,
+    };
+
+    const response = await context.db.LossPayeeFinancialUk.updateOne({
+      where: {
+        id,
+      },
+      data: updateData,
+    });
+
+    if (response) {
+      return {
+        success: true,
+      };
+    }
+
+    return {
+      success: false,
+    };
+  } catch (err) {
```

Comment:
Are you using `error` or `err` in EXIP for catch variable name please?

Reply:
err

Reply:
Yes we're using `err`, I thought this was the standard/pattern we want to have.

Otherwise @abhi-markan  let us know which is preferred, we can create a tech ticket to changae this throughout the codebase.

Reply:
`error` will be my preference as we are using this in DTFS and APIM but not a blocker. Thanks